app/jobs/replicated_file_check_job.rb

```diff
@@ -10,13 +10,24 @@ class ReplicatedFileCheckJob < ApplicationJob
 
   # @param [PreservedCopy] verify that the archived preserved_copy exists on an endpoint
   def perform(preserved_copy)
-    return if preserved_copy.unreplicated? # FIXME: storytime (Monday 07/02)
+    if preserved_copy.unreplicated?
+      Rails.logger.error("#{preserved_copy} should be replicated, but has a status of 'unreplicated'.")
```

Comment:
Should this pull from the actual status, instead of using a string?

Reply:
yeah i'm not sure. since we know its `unreplicated` i just added it, i can interpolate if you'd like. What do you think?

Reply:
I'd prefer interpolation, in the unlikely case we decide to change the name of the status at the constant level.

Reply:
sounds good. just pushed.